models.py

```diff
@@ -0,0 +1,186 @@
+from flask_sqlalchemy import SQLAlchemy
```

Comment:
Let's break these models out into individual model files.

something like this
![screen shot 2019-01-28 at 10 26 05 pm](https://user-images.githubusercontent.com/903532/51888463-df74bd80-234b-11e9-88aa-07642cd26fc9.png)